handsontable/src/plugins/autoRowSize/autoRowSize.js

```diff
@@ -722,6 +724,8 @@ export class AutoRowSize extends BasePlugin {
   #onInit() {
     this.recalculateAllRowsHeight();
     this.#isInitialized = true;
+
+    addClass(this.hot.rootElement, 'htAutoRowSize');
```

Comment:
Can I suggest moving that line to the `enablePlugin` method? I think it would improve the readability of the code by cooperating with the `disablePlugin` method, where the CSS class is removed.

Also, I'd move the `htAutoRowSize` string to const.